lib/tasks/migrate_users.rake

```diff
@@ -39,6 +41,26 @@ namespace :otis do
         unless Services.checkpoint.permits?(agent, view_role, res_inst)
           Services.checkpoint.grant!(agent, view_role, res_inst)
         end
+        unless Services.checkpoint.permits?(agent, view_role, res_contact)
+          Services.checkpoint.grant!(agent, view_role, res_contact)
+        end
+        unless Services.checkpoint.permits?(agent, view_role, res_contact_type)
+          Services.checkpoint.grant!(agent, view_role, res_contact_type)
+        end
+      end
+    end
+    if Otis.config.contact.present?
```

Comment:
The names of the keys in `Otis.config` for permissions have become a little bit confusing. Essentially there should be four roles now:

- superuser (should be granted admin all)
- staff (should be granted view all)
- institution admin (should be granted admin institutions)
- contact admin (should be granted admin contacts, admin contact type)

We may want to start making these roles more explicit in checkpoint but I would need to think about exactly how we do that.

Reply:
I looked into "policy" objects in the Chipmunk repo (and also in Heliotrope but it's a lot more complicated) and I agree we are pushing up against maintainability limits in the way we do roles. But, decided it was enough out of scope for this round that can-kicking-down-the-road was called for.

Reply:
:+1: